src/frontend/src/handler/create_source.rs

```diff
@@ -1465,6 +1466,14 @@ pub fn bind_connector_props(
                 .to_string(),
         );
     }
+    if !is_create_source && with_properties.is_mysql_cdc_connector() {
```

Comment:
Should put in `if is_create_source && create_cdc_source_job {` branch, cdc Table doesn't need this option.

Reply:
:eyes: https://github.com/risingwavelabs/risingwave/pull/13031 This PR is only for CDC table, but not (shared) CDC source.

Reply:
Do you think it should apply to both `is_create_source` and `! is_create_source`?

Reply:
> 👀 #13031 This PR is only for CDC table, but not (shared) CDC source.

You are right. This for the old syntax previously, I think we can apply to both `is_create_source` and ! `is_create_source`.